cmd/addvalidators.go

```diff
@@ -236,6 +249,34 @@ func writeClusterManifest(filename string, cluster *manifestpb.Cluster) error {
 	return nil
 }
 
+// writeManifestBackup writes the provided cluster in a cluster manifest backup file inside the backup directory.
+// The backup files are stored as "cluster-manifest-backup-<unix_timestamp>.pb".
+func writeManifestBackup(backupDir string, cluster *manifestpb.Cluster) error {
+	// Check if the backup directory exists, creating a new one if it doesn't exist.
+	info, err := os.Stat(backupDir)
+	if errors.Is(err, fs.ErrNotExist) {
+		err := os.Mkdir(backupDir, 0o755)
+		if err != nil {
+			return errors.Wrap(err, "create backup dir", z.Str("backup-dir", backupDir))
+		}
+	} else if err != nil {
+		return errors.Wrap(err, "error retrieving backup dir info", z.Str("backup-dir", backupDir))
+	} else if !info.IsDir() {
+		return errors.New("backup dir already exists and is a file", z.Str("backup-dir", backupDir))
+	}
+
+	filename := fmt.Sprintf("cluster-manifest-backup-%d.pb", time.Now().Unix()) // Ex: "cluster-manifest-backup-1687250009.pb"
```

Comment:
#2345  says:

> Each backup file should be named with a lexicographic datetime (fancy way of saying reverse order so it sorts well). Timestamps in unix time aren't user friendly despite being sortable. cluster-manifest.pb.20230721183000"

Is this still the case?

Reply:
Probably yes, since the ticket has been edited now. I'll make this file use `YYYYMMDDHHMMSS` format, will update my PR in a bit